### PR TITLE
Allow rails cops to be required separately

### PR DESCRIPTION
**Pull request by JoeSouthan**

### CODE REVIEW

File: gc_ruboconfig.gemspec [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 Gem::Specification.new do |spec|
   spec.name          = 'gc_ruboconfig'
-  spec.version       = '2.33.0'
+  spec.version       = '2.34.0'
```
Comment (kevinrobayna) [inline gc_ruboconfig.gemspec:5]: this is breaking change (if you update the version you won't get any of the rails setup and you need to manually configure it)

**Review by kevinrobayna — CHANGES_REQUESTED** [review]
please update the version type we will be release as it's not a minor change.

File: gc_ruboconfig.gemspec (hunk 1 above)
Comment (JoeSouthan) [reply gc_ruboconfig.gemspec]: Done!

**Comment by ivgiuliani** [comment]
Not sure this is actually working as intended, I think we need to remove the `rubocop-rails` require at the top of rubocop.yml too? Otherwise we just get the default settings for anything `Rails/*`

**Comment by JoeSouthan** [comment]
Ah fair, will fix

**Pull request merged**